Bottom: 4cbdc23a02a619e2cda3cc670fc42bd04bb2671c
Top:    e043f69a76b3e8277146c058fed950c44011ee56
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-04 20:08:16 +0000

Refresh of add-a-load-more-hacky-models-f

---

diff --git a/src/hid/common/hidgl_package_acy_resistor.c b/src/hid/common/hidgl_package_acy_resistor.c
index ecc881b..4883778 100644
--- a/src/hid/common/hidgl_package_acy_resistor.c
+++ b/src/hid/common/hidgl_package_acy_resistor.c
@@ -378,12 +378,12 @@ static void
 setup_resistor_2300mil_texture (ElementType *element, GLfloat *res_body_color)
 {
   GLfloat tex_data[32][3];
-  int strip;
+  int texel;
 
-  for (strip = 0; strip < sizeof (tex_data) / sizeof (GLfloat[3]); strip++) {
-    tex_data[strip][0] = res_body_color[0];
-    tex_data[strip][1] = res_body_color[1];
-    tex_data[strip][2] = res_body_color[2];
+  for (texel = 0; texel < sizeof (tex_data) / sizeof (GLfloat[3]); texel++) {
+    tex_data[texel][0] = res_body_color[0];
+    tex_data[texel][1] = res_body_color[1];
+    tex_data[texel][2] = res_body_color[2];
   }
 
   glTexImage1D (GL_TEXTURE_1D, 0, GL_RGB, 32, 0, GL_RGB, GL_FLOAT, tex_data);
@@ -393,12 +393,12 @@ static bool
 setup_hvdiode_texture (ElementType *element, GLfloat *body_color)
 {
   GLfloat tex_data[32][3];
-  int strip;
+  int texel;
 
-  for (strip = 0; strip < sizeof (tex_data) / sizeof (GLfloat[3]); strip++) {
-    tex_data[strip][0] = body_color[0];
-    tex_data[strip][1] = body_color[1];
-    tex_data[strip][2] = body_color[2];
+  for (texel = 0; texel < sizeof (tex_data) / sizeof (GLfloat[3]); texel++) {
+    tex_data[texel][0] = body_color[0];
+    tex_data[texel][1] = body_color[1];
+    tex_data[texel][2] = body_color[2];
   }
 
   /* Draw the stripe */
@@ -406,7 +406,6 @@ setup_hvdiode_texture (ElementType *element, GLfloat *body_color)
 
   glTexImage1D (GL_TEXTURE_1D, 0, GL_RGB, 32, 0, GL_RGB, GL_FLOAT, tex_data);
 
-  free (stripes);
   return false;
 }
